interfaces/builtin/anbox_support.go

```diff
@@ -0,0 +1,512 @@
+    mount fstype=tmpfs -> /var/snap/@{SNAP_NAME}/common/lxc/**/,
+    mount fstype=proc -> /var/snap/@{SNAP_NAME}/common/lxc/proc/,
+    mount options=(ro, nosuid, nodev, noexec, remount, bind) -> /var/snap/@{SNAP_NAME}/common/lxc/proc/sys/{,**/},
+    mount options=(rw, move) /var/snap/@{SNAP_NAME}/common/lxc/**/ -> /var/snap/@{SNAP_NAME}/common/lxc/**/,
+    mount options=(rw, nosuid, nodev, noexec) -> /var/snap/@{SNAP_NAME}/common/lxc/sys/,
+    mount options=(ro, nosuid, nodev, noexec, remount, bind) -> /var/snap/@{SNAP_NAME}/common/lxc/sys/,
+    mount options=(rw, nosuid, nodev, noexec, remount, bind) -> /var/snap/@{SNAP_NAME}/common/lxc/sys/{,**/},
+    mount fstype=sysfs -> /var/snap/@{SNAP_NAME}/common/lxc/**/,
+
+    mount options=(rw, bind) /{,var/snap/@{SNAP_NAME}/common/lxc/}dev/full -> /var/snap/@{SNAP_NAME}/common/lxc/dev/full,
+    mount options=(rw, bind) /{,var/snap/@{SNAP_NAME}/common/lxc/}dev/null -> /var/snap/@{SNAP_NAME}/common/lxc/dev/null,
+    mount options=(rw, bind) /{,var/snap/@{SNAP_NAME}/common/lxc/}dev/random -> /var/snap/@{SNAP_NAME}/common/lxc/dev/random,
+    mount options=(rw, bind) /{,var/snap/@{SNAP_NAME}/common/lxc/}dev/urandom -> /var/snap/@{SNAP_NAME}/common/lxc/dev/urandom,
+    mount options=(rw, bind) /{,var/snap/@{SNAP_NAME}/common/lxc/}dev/tty -> /var/snap/@{SNAP_NAME}/common/lxc/dev/tty,
+    mount options=(rw, bind) /{,var/snap/@{SNAP_NAME}/common/lxc/}dev/zero -> /var/snap/@{SNAP_NAME}/common/lxc/dev/zero,
+    mount options=(rw, bind) /{,var/snap/@{SNAP_NAME}/common/lxc/}dev/pts/* -> /var/snap/@{SNAP_NAME}/common/lxc/dev/pts/*,
+    mount options=(rw, bind) /{,var/snap/@{SNAP_NAME}/common/lxc/}dev/pts/0 -> /var/snap/@{SNAP_NAME}/common/lxc/dev/console,
+
+    pivot_root /var/snap/@{SNAP_NAME}/common/lxc/ -> /var/snap/@{SNAP_NAME}/common/lxc/,
+
+    # post pivot_root
+    # @{HOME} doesn't work here
+    mount options=(rw, bind) /home/*/snap/@{SNAP_NAME}/** -> /var/snap/@{SNAP_NAME}/common/lxc/dev/**,
+    mount options=(rw, bind) /home/*/snap/@{SNAP_NAME}/** -> /var/snap/@{SNAP_NAME}/common/devices/*,
+    mount options=(rw, bind) /var/snap/@{SNAP_NAME}/common/devices/* -> /var/snap/@{SNAP_NAME}/common/lxc/dev/*,
+    mount fstype=devpts options=(rw, nosuid, noexec) -> /dev/pts/,
+    mount options=(rw, bind) /dev/{,pts/}ptmx -> /dev/{,pts/}ptmx,
+
+    # apparmor="DENIED" operation="mount" info="failed mntpnt match" error=-13 profile="snap.anbox.container-manager//lxc" name="/" pid=4539 comm="anbox" flags="rw, rslave"
+    umount /,
+    umount /var/snap/@{SNAP_NAME}/common/lxc/dev/console,
+    # done post pivot_root
+
+    # starting processes in the container
+    /anbox-init.sh ixr,
+    @{PROC}/1/attr/current r,
+
+    # end processes in the container
+
+    owner @{PROC}/@{pid}/net/unix r,
+    unix (bind,listen) type=stream addr="@/var/snap/anbox/common/containers/default/command",
+    /sys/fs/cgroup/*/lxc/{,**} rw,   # this should be anbox specific
```

Comment:
Please make this anbox specific otherwise anbox might interfere with other lxcs on the system.